scripts/kind-build-test.sh

```diff
@@ -217,6 +217,16 @@ for crd_file in $service_config_dir/crd/common/bases; do
 done
 echo "ok."
 
+echo -n "creating ack-system namespace"
```

Comment:
One questions:
1) `deployment.yaml` file in kustomize tries to `kubectl apply` `ack-system` namespace again? Did you any error/warning for that when running test locally?

Reply:
Kustomize doesn't care - it just overwrites it with the same value.

Reply:
@vijtrip2 What @RedbackThomson said is correct, since this is creating the namespace the same way as in the `deployment.yaml` there aren't any issues/warnings/errors. Below are the logs for clarity, meant to add them to the PR but forgot.

```
creating ack-system namespacenamespace/ack-system created
loading RBAC manifests for s3 into the cluster ... ok.
loading service controller Deployment for s3 into the cluster ...ok.
```